### PR TITLE
Release/1.1.2

### LINKED ISSUE #185: Release version 1.1.2
This issue is for tracking changes for the 1.1.2 release. Target release date: October 2023.

### [Release steps](https://github.com/10up/maps-block-apple/blob/develop/CONTRIBUTING.md#release-instructions)
- [x] Branch: Starting from `develop`, cut a release branch named `release/1.1.2` for your changes.
- [x] Version bump: Bump the version number in `maps-block-apple.php`, `readme.txt`, `package.json` and `package-lock.json` if it does not already reflect the version being released.  Update both the plugin "Version:" property and the plugin `MAPS_BLOCK_APPLE_VERSION` constant in `maps-block-apple.php`.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk`, ensuring you pull the most recent changes into `develop` first (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`). `trunk` contains the stable development version.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] [Compare](https://github.com/10up/maps-block-apple/compare/trunk...develop) trunk to develop to ensure no additional changes were missed.
- [x] Test the pre-release ZIP locally by [downloading](https://github.com/10up/maps-block-apple/actions/workflows/build-release-zip.yml) it from the Build release zip action artifact and installing it locally. Ensure this zip has all the files we expect, that it installs and activates correctly and that all basic functionality is working.
- [x] Release: Create a [new release](https://github.com/10up/maps-block-apple/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch. Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the [closed issues on the milestone](https://github.com/10up/maps-block-apple/milestone/10?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/maps-block-apple/actions/workflows/deploy-to-wordpress-org.yml) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on [WordPress.org](https://wordpress.org/plugins/maps-block-apple/). This may take a few minutes.
- [x] Close the milestone: Edit the [1.1.1 milestone](https://github.com/10up/maps-block-apple/milestone/10) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.1.2` do not make it into the release, update their milestone to `1.2.0` or `Future Release`.

### PR DESCRIPTION
**Pull request by dkotter**
### Description of the Change

Version bump and prep for the 1.1.2 release.

Closes #185 

### How to test the Change

Verify all file changes look accurate

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -72,6 +72,18 @@ Yes, you can track your MapKit JS useage on the [MapKit JS Developer Dashboard](
 
 == Changelog ==
```
Comment (peterwilsoncc) [inline readme.txt:73]: The changelog is getting a little large, let's keep the last two or three releases in the wp.org readme and refer people to the GitHub changelog for the full log.

Linking to a changelog is suggested by [the plugin readme docs](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#file-size), so won't fall fowl of the plugin guidelines.

**Review by peterwilsoncc — COMMENTED** [review]
LGTM but one suggestion to reduce the size of the readme.txt file.

File: readme.txt (hunk 1 above)
Comment (dkotter) [reply readme.txt:73]: Updated 2549d61

**Review by jeffpaul — APPROVED** [review]
Looks good, thanks for handling the release!

**Pull request merged**